src/js/Controllers/UIController.js

```diff
@@ -381,6 +443,9 @@ class UIController {
         clearTimeout(this.timers[alert.msgID])
         delete this.timers[alert.msgID]
 
+        let imageValidationSuccess = GetValidationResult(alert.msgID);
+        RemoveImageValidationRequest(alert.msgID)
```

Comment:
I feel like these methods should be combined some how, do you think maybe Get should call Remove?

Reply:
Agreed. Made the change in https://github.com/smartdevicelink/generic_hmi/pull/314/commits/b14ac9f32393890425af94b6ab463678f8bc728a. Let  me know if that looks good